ocimotel/repoclient.go

```diff
@@ -30,18 +35,52 @@ func NewOciRepo(ref *ociMotelReference) (r OciRepo, err error) {
 	if ref.port != -1 {
 		port = fmt.Sprintf("%d", ref.port)
 	}
-	r.url = fmt.Sprintf("http://%s:%s", server, port)
-	queryURI := fmt.Sprintf("%s/v2/", r.url)
-	client := &http.Client{}
-	resp, err := client.Get(queryURI)
-	if err != nil {
-		return r, err
+
+	insecureSkipVerify := (sys.DockerInsecureSkipTLSVerify == types.OptionalBoolTrue)
```

Comment:
Seems like we should just add our own, vs. piggybacking on docker's?

Reply:
Yes, SystemContext does have OCI* equivalent fields, except skopeo appears to use the Docker* ones. Should that change, this is really the only place to update it.

Reply:
Can you elaborate a bit? skopeo definitely sets the OCI version of the fields too: https://github.com/containers/skopeo/blob/master/cmd/skopeo/utils.go#L178

I think it probably makes sense to have our own in our own namespace.

Reply:
https://github.com/containers/skopeo/blob/master/cmd/skopeo/utils.go#L115

Reply:
> I think it probably makes sense to have our own in our own namespace.
Yes, step 2?

Reply:
Yes, I see both OCI and Docker options in that list?

Reply:
> Yes, I see both OCI and Docker options in that list?

Correct, however, those fields are not set *by skopeo*
For example, although we have:
https://github.com/containers/image/blob/master/oci/layout/oci_src.go#L35

skopeo still sets it so.
https://github.com/containers/skopeo/blob/master/cmd/skopeo/utils.go#L128

grok'ing skopeo code for "OCIInsecureSkipTLSVerify" (vendor/ doesn't count) returns empty.

Like you said, long-term solution is to drive from command-line parsing with our own namespace.